webapp/src/lib/LeafletParcelGrid.js

```diff
@@ -169,14 +171,24 @@ export const LeafletParcelGrid = L.Layer.extend({
     // Clear canvas
     const ctx = this.canvas.getContext('2d')
     ctx.clearRect(0, 0, this.canvas.width, this.canvas.height)
-
-    for (let index = tiles.length - 1; index >= 0; index--) {
-      this.renderTile(tiles[index])
+    let marker = null
+    for (let index = 0; index < tiles.length; index++) {
+      const point = this.renderTile(tiles[index])
+      if (point) {
```

Comment:
_To consider_

I think renderTile is doing two things at the same time and it's a bit confusing. I wouldn't have guessed that the return value signifies that a marker is present.
I know it's a bit less performant but my first instinct is to do something like:

```javascript
for (let index = 0; index < tiles.length; index++) {
  const tile = tiles[index]
  const tileId = this.renderTile(tile)
  if (this.options.marker === tileId) {
    marker = this.map.latLngToContainerPoint(tile.center)
  }
// (...)

renderTile(tile) {
  // (...)
  return id
},
```

What do you think?